snapcraft.yaml

```diff
@@ -63,6 +63,7 @@ plugs:
   dot-thunderbird:
     interface: personal-files
     read: [$HOME/.thunderbird]
```

Comment:
I believe this line can be removed. With write access, there is automatically read access.

Reply:
Thanks for chiming in, that's true:

> write (plug): list of files and/or directories for read/write access

— https://forum.snapcraft.io/t/the-personal-files-interface/9357

I already heard that about Apparmor profiles earlier just last week and forgot it. I'm too attached to old school Unix permissions. (: